text/0000-edition-2021-and-beyond.md

```diff
@@ -0,0 +1,194 @@
+- Feature Name: N/A
+- Start Date: 2020-07-29
+- RFC PR: [rust-lang/rfcs#0000](https://github.com/rust-lang/rfcs/pull/0000)
+- Rust Issue: [rust-lang/rust#0000](https://github.com/rust-lang/rust/issues/0000)
+
+# Summary
+[summary]: #summary
+
+* Announce plans for a Rust 2021 Edition, and for a regular cadence of editions every 3 years thereafter.
+    * We will roll out an edition regardless of whether there are breaking changes.
+* Unlike Rust 2018, we will avoid using editions as a "deadline" to tie together high-priority projects.
+    * Instead, we embrace the train model, but editions are effectively a "somewhat bigger release", giving us an opportunity to give an overview of all the work that has landed over the previous three years.
+* We specify a cadence for Edition lints.
+    * "Edition idiom" lints for Edition N will warn for editions before N, and become "deny by default" in Edition N.
+    * Since it would be disruptive to introduce deny-by-default lints for Rust 2018 now, the Rust 2018 lints are repurposed into Rust 2021 Edition lints.
+* We specify a policy on reserving keywords and other prospective changes.
+    * In short, reserving keywords is allowed only as part of an active project group.
```

Comment:
Ideally the answered are numbered for easy relation to the questions (1.-5.).
I do count 4 answers and I guess you missed "How do we handle macros, particularly procedural macros, that may mix source from multiple editions?".